runtime/doc/repeat.txt

```diff
@@ -1048,6 +1048,9 @@ For example, to profile the one_script.vim script file: >
 		If {fname} already exists it will be silently overwritten.
 		The variable |v:profiling| is set to one.
 
+:prof[ile] stop
+               Write the logfile and stop profiling.
```

Comment:
I have not tried yet, but what happens if I call `:profile stop` twice in a row?

I also see that behavior of calling `:profile start …` twice is not documented (already the case prior to your change).

Also what happens if we call `:profile dump` without prior call to `:profile start …`?